pkg/kubectl/cmd/expose.go

```diff
@@ -61,7 +61,8 @@ func NewCmdExposeService(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().String("protocol", "TCP", "The network protocol for the service to be created. Default is 'tcp'.")
 	cmd.Flags().Int("port", -1, "The port that the service should serve on. Required.")
 	cmd.MarkFlagRequired("port")
-	cmd.Flags().Bool("create-external-load-balancer", false, "If true, create an external load balancer for this service. Implementation is cloud provider dependent. Default is 'false'.")
```

Comment:
I did (I think you did, actually); it just moved down a line!  And I think we continue to pass create-external-load-balancer to the helper, which converts it to a visibility flag.